boa_engine/src/bytecompiler.rs

```diff
@@ -2346,4 +2376,414 @@ impl<'b> ByteCompiler<'b> {
         }
         Ok(has_identifier_argument)
     }
+
+    fn class(&mut self, class: &Class, expression: bool) -> JsResult<()> {
```

Comment:
Could we add a bit of documentation here?

Reply:
Done